x-pack/plugins/enterprise_search/public/applications/shared/constants/access.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const NO_ACCESS = {
```

Comment:
nitpick: Giving this an explicit ProductAccess type would help catch a few errors during development.